src/style.css

```diff
@@ -0,0 +1,131 @@
+.body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
+  .body {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+}
+
+@media (min-width: 900px) {
+  .body {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+}
+
+@media (min-width: 1200px) {
+  .body {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  
+  .block {
+    width: 33.3%;
+  }
+}
+
+.block {
+  display: flex;
+  height: 300px;
+  min-width: 300px;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
+  .block {
+    flex-grow: 1;
+  }
+}
```

Comment:
you can all selectors that you want to style inside this media query, no need to write another one like in line 73,89 105 and 121